daemons/pacemakerd/pacemakerd.c

```diff
@@ -247,8 +264,7 @@ main(int argc, char **argv)
 
     rc = pcmk_new_ipc_api(&old_instance, pcmk_ipc_pacemakerd);
     if (old_instance == NULL) {
-        fprintf(stderr, "Could not connect to pacemakerd: %s",
-                pcmk_rc_str(rc));
+        out->err(out, "Could not connect to pacemakerd: %s", pcmk_rc_str(rc));
```

Comment:
While we're here, let's change this message to something like "Could not check for existing pacemakerd:" since it's not an error if we can't connect (that just means there's no existing instance).

Reply:
Done